### PR TITLE
Feature test integration

### PR DESCRIPTION
**Pull request by azy64**
This pull request is about creating:
- Splash screen
- Sign up and login pages
- Home page (categories page)
- Transactions page
- "Add a new category" page
- "Add a new transaction" page
- Create unit and integration tests for all the most important components of your RoR application.
And here is my presentation link:
https://www.loom.com/share/c111450efa6744c39fb133b3d8b1023c

### CODE REVIEW

File: spec/features/entity_spec.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    click_link @group.name
+  end
+
+  scenario 'show group\'s name on page' do
+    expect(page).to have_content @group.name
+  end
+
+  scenario 'show the transaction\'s amount on page' do
+    expect(page).to have_content @entity.amount
+    # expect(page).to have_content 'ppp.png'
```
Comment (arslanbisharat) [inline spec/features/entity_spec.rb]: - [ ] Please remove the commented code

File: spec/features/group_spec.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    visit groups_path
+  end
+
+  scenario 'show My groups on page' do
+    expect(page).to have_content 'GROUPS'
+  end
+
+  scenario 'show group details on page' do
+    expect(page).to have_content @group.name
+    # expect(page).to have_content 'ppp.png'
```
Comment (arslanbisharat) [inline spec/features/group_spec.rb]: - [ ]  Please remove the commented code

File: spec/models/entity_spec.rb [hunk 3]
```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe Entity, type: :model do
+  it 'valid if name is present' do
+    @user = User.create!(name: 'Henry', email: 'henry@gmail.com', password: 'kkkkkk')
+    @group = @user.groups.create(name: 'ADIDAS6GROUP', icon: 'grams.png')
+    @entity = @group.entities.create(name: 'Chapo', amount: 30.0, user: @user)
+    # @entity = @user.foods.create(name: 'Chapo', measurement_unit: 'grams', price: 150)
```
Comment (arslanbisharat) [inline spec/models/entity_spec.rb]: - [ ]  Please remove the commented code

File: README.md [hunk 4]
```diff
@@ -1,24 +1,95 @@
...
 
-* Database creation
+Additional description about the project and its features.
 
-* Database initialization
+## Built With
+Ruby 
+Ruby on Rails
 
-* How to run the test suite
```
Comment (arslanbisharat) [inline README.md:18]: - [ ] Tests are missing from the project. 


- Make sure the app has unit tests for all the most important components of the RoR application.

- Make sure the app has integration tests for all the most important components of the RoR application.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: README.md (hunk 4 above)
Comment (azy64) [reply README.md:18]: are you sure that tests are missing? which tests ?

File: spec/features/entity_spec.rb (hunk 1 above)
Comment (DeliceLydia) [reply spec/features/entity_spec.rb]: Fixed

File: spec/features/group_spec.rb (hunk 2 above)
Comment (DeliceLydia) [reply spec/features/group_spec.rb]: Fixed

File: spec/models/entity_spec.rb (hunk 3 above)
Comment (DeliceLydia) [reply spec/models/entity_spec.rb]: Resolved

**Review by DeliceLydia — APPROVED** [review]
# Approved 🎉🎉 

## Hi @azy64 , 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


### Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**